library.c

```diff
@@ -1065,6 +1065,15 @@ int redis_cmd_append_sstr_i64(smart_string *str, int64_t append) {
     return redis_cmd_append_sstr(str, nbuf, len);
 }
 
+/*
+ * Append a 64-bit unsigned integer to our command
+ */
+int redis_cmd_append_sstr_ui64(smart_string *str, uint64_t append) {
+    char nbuf[64];
```

Comment:
```suggestion
    char nbuf[21];
```

strlen(18446744073709551615) = 20 + 1 for '\0'

Reply:
@michael-grunder nice! I reduced `redis_cmd_append_sstr_i64` to `21` too.

The longest number it can represent is `-9223372036854775808` (including the sign is 20 characters) + `\0` I believe.